src/script/lua_api/l_mainmenu.cpp

```diff
@@ -385,7 +385,8 @@ int ModApiMainMenu::l_get_content_info(lua_State *L)
 	if (spec.type == "mod") {
 		ModSpec spec;
 		spec.path = path;
-		parseModContents(spec);
+		// TODO return nothing on failure (needs callers to handle it)
+		static_cast<void>(parseModContents(spec));
```

Comment:
I fear this TODO will still be here in 2 years

Reply:
So do I. That's why I normally save source file space by omitting it.

Reply:
Removing both the comment and the void cast would also be fine.

Reply:
I'll leave them in and try to finish them this week, so that they won't be there 2 years from now.